plugins/PolylangPro.php

```diff
@@ -44,6 +44,14 @@ public function getDownloadUrl() {
 			'url'        => getenv( 'POLYLANG_PRO_URL' ),
 			'version'    => $this->version,
 		) ), true );
+
+		/*
+		 * If the response is not the version we asked for then, bail
+		 */
+		if ( $response['stable_version'] !== $this->version ) ) {
+			return '';
+		}
```

Comment:
Fix syntax error (extra `)`), replace `stable_version` with `new_version` according to EDD SL API, and ensure `new_version` exists.

```suggestion
		/**
		 * If the response does not have a version number or the version number
		 * does not match the package constraint, bail.
		 */
		if ( empty( $response['new_version'] ) || $response['new_version'] !== $this->version ) {
			return '';
		}
```